### PR TITLE
[www] Fix quotes, fixes „Invalid date“ of latest blog post

### PR DESCRIPTION
**Pull request by fk**

### CODE REVIEW

File: docs/blog/2018-2-16-how-to-build-a-website-with-react/index.md [hunk 1]
```diff
@@ -29,28 +26,29 @@ React is not picky; it is happy to be used in only parts of your site, so you ca
...
-* *Solving performance problems:* Gatsby sites rarely has performance problems due to Gatsby’s way of loading static files.
+
+* _Pulling in data:_ In Gatsby, GraphQL and plugins help you use data from nearly any source (including both traditional CMSs and headless CMSs. Some people I’ve talked to recently even built their Gatsby sites with Google sheets as the data source.
+* _Creating pages and routes:_ Gatsby also gives you an intuitive interface for creating pages and routes. So intuitive, in fact, that when I talked to a coworker, I said, “I remember creating pages and links to those pages from other pages, but I don’t remember creating any routes in Gatsby.” They responded, “Yeah, Gatsby took care of that for you.”
+* _Solving performance problems:_ Gatsby sites rarely has performance problems due to Gatsby’s way of loading static files.
```
Comment (m-allanson) [inline docs/blog/2018-2-16-how-to-build-a-website-with-react/index.md]: Small typo - 'has' should be 'have'

Comment (m-allanson) [reply docs/blog/2018-2-16-how-to-build-a-website-with-react/index.md]: Oh maybe I can do this change through GitHub - I'll try it now.

Comment (m-allanson) [reply docs/blog/2018-2-16-how-to-build-a-website-with-react/index.md]: It worked!

**Comment by KyleAMathews** [comment]
Thanks @fk and @m-allanson!

File: docs/blog/2018-2-16-how-to-build-a-website-with-react/index.md [hunk 2]
```diff
@@ -29,28 +26,29 @@ React is not picky; it is happy to be used in only parts of your site, so you ca
...
-    * Title
-    * add a to-do line
-    * to-do line
-        * subtasks within to-dos
-    * show completed to-dos button
+  * Title
+  * add a to-do line
+  * to-do line
+    * subtasks within to-dos
+  * show completed to-dos button
```
Comment (fk) [inline docs/blog/2018-2-16-how-to-build-a-website-with-react/index.md]: Hey @KyleAMathews and @m-allanson, sorry for not merging this myself, got sidetracked a little. :-/

What I wanted to check before merging is what I just checked now: I have Prettier(-eslint) set up to run everytime I save, thus all the reformatting around here w/o explicitly running `format-markdown`. Before merging I noticed that the list above doesn't display correctly after Prettier changed indention from four to two spaces (after/before with four spaces): 

image

image

Will take a look at `using-remark` and open an issue.

**Pull request merged**